src/control/lib/support/README.md

```diff
@@ -0,0 +1,76 @@
+# support collect-log command
+
+Support collect-log command will collect the logs on the servers or individual clients
+for debugging purpose.This options is available for `daos_server`, `dmg` and `daos_agent` binaries. 
+It will collect the specific logs, config and other DAOS related metrics and system information.
+
+`dmg support collect-log` is the single command, which will initiate the log collection 
```

Comment:
```suggestion
for debugging purpose.This options is available for `daos_server`, `dmg` and `daos_agent` binaries.
It will collect the specific logs, config and other DAOS related metrics and system information.

`dmg support collect-log` is the single command, which will initiate the log collection
```